Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
 name = "varinteger"
-version = "1.0.7-alpha.0"
+version = "2.0.0"
 description = "Rust module for encoding/decoding varints that doesn't do any IO. Inspired by the Node.js varint module"
-authors = ["Mathias Buus <mathiasbuus@gmail.com>"]
-repository = "https://github.com/mafintosh/varinteger-rs"
+authors = ["James Halliday", "Karissa McKelvey <karissa.mckelvey@gmail.com>", "Mathias Buus <mathiasbuus@gmail.com>"]
+repository = "https://github.com/datrs/varinteger"
 keywords = ["protobuf", "varint", "variable", "integer"]
 readme = "README.md"
 license = "MIT"
+
+[dependencies]
+failure = "0.1.8"
```

Comment:
The `failure` crate is kinda deprecated IIRC, see [this comment by the maintainer](https://internals.rust-lang.org/t/failure-crate-maintenance/12087). For a simple custom error as we need here, I'd propose to just [implement the Error trait](https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/define_error_type.html) on a `DecodeError` struct.